packages/shared/routes/setup/Password.svelte

```diff
@@ -66,6 +67,16 @@
     function handleBackClick(): void {
         $appRouter.previous()
     }
+
+    function checkPasswordStrength(password: string): any {
+        const NUMBER_OF_STRENGTH_VALIDATION_CHARS = 64
+        const limitedPassword = password.substring(0, NUMBER_OF_STRENGTH_VALIDATION_CHARS - 1)
+        const hasCheckedPasswordChanged = lastCheckedPassword !== limitedPassword
+        if (hasCheckedPasswordChanged) {
+            lastCheckedPassword = limitedPassword
+            return zxcvbn(limitedPassword)
+        }
```

Comment:
If it returns undefined, it finds no translation with key GENERAL.PASSWORDSTRENGTHUNDEFINED